src/ddmd/complex_t.h

```diff
@@ -22,7 +22,7 @@ struct complex_t
     real_t re;
     real_t im;
 
-    complex_t(real_t re) : re(re), im(ldouble(0)) {}
+    complex_t(real_t re) : re(re), im(0) {}
```

Comment:
Why don't you implement `ldouble()`?

Reply:
Because it's ugly as hell and a `real_t` constructor accepting 0 makes much more sense. I never understood why you readded it.

Reply:
real_t is used in a union, and so must be a POD.  You can't add any constructors to the type.

Reply:
That was pre-C++11.

Reply:
It is still too soon to be breaking C++98 compatibility.

Reply:
[LDC doesn't care and requires C++11.]

Reply:
You could initialize it with the constant CTFloat::zero.

(GCC requires C++98 for bootstrapped builds)